src/UI/class-settings-page.php

```diff
@@ -81,13 +81,34 @@ final class Settings_Page {
 	 */
 	private $hook_suffix;
 
+	/**
+	 * Options for badges that are displayed for managed options.
+	 *
+	 * @since 3.9.0
+	 *
+	 * @var array<string, mixed>
+	 */
+	private $managed_options_badge = array();
+
 	/**
 	 * Constructor.
 	 *
 	 * @param Parsely $parsely Instance of Parsely class.
 	 */
 	public function __construct( Parsely $parsely ) {
 		$this->parsely = $parsely;
+
+		$managed_options_badge = apply_filters(
+			'wp_parsely_managed_options_badge',
+			array(
+				'text' => __( 'Upgrade', 'wp-parsely' ),
+				'url'  => 'https://www.parse.ly/getdemo/',
```

Comment:
Note that when we start to use the integrations framework for joint customers as well, this will not make much sense. We'll need to remember to use the filter to handle P100 vs Joint users.

Reply:
Started a private communication thread about this.